src/internal.c

```diff
@@ -4922,6 +4922,9 @@ int EccSign(WOLFSSL* ssl, const byte* in, word32 inSz, byte* out,
         }
         ret = ssl->ctx->EccSignCb(ssl, in, inSz, out, outSz, keyBuf,
             keySz, ctx);
+        if (ret == CRYPTOCB_UNAVAILABLE) {
```

Comment:
In other places where we fell back to software crypto, we only did so when `WOLFSSL_RENESAS_SCEPROTECT` or `WOLFSSL_RENESAS_TSIP_TLS` were defined.

I think we should follow the existing pattern and only fall back like this for those use cases, not generally.

Reply:
Agreed and added macro guard.